tenseal/cpp/tensors/ckkstensor.cpp

```diff
@@ -76,7 +76,37 @@ shared_ptr<CKKSTensor> CKKSTensor::power_inplace(unsigned int power) {
 
 shared_ptr<CKKSTensor> CKKSTensor::add_inplace(
```

Comment:
The PR should inplace the add_plain implementation too?

Reply:
Yeah, as well as sub and mul. I just wanted to have a first review since other methods would look similar.